integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5159,6 +5177,36 @@ msgstr "Bild/PNG"
 msgid "Table Document/CSV"
 msgstr "Tabellendokument/CSV"
 
+#: cms/templates/tutorials/page_tree.html:9
+msgid "Introduction - Page Tree"
+msgstr "Einführung in den neuen Seiten-Baum"
+
+#: cms/templates/tutorials/page_tree.html:13
+msgid ""
+"In our new tree view, we have merged the two previous views into one which "
```

Comment:
`We merged "All pages" and "Tree View" into one single view. In this view, you can do the following:`